plugins/hypervisors/ovm3/src/main/java/com/cloud/hypervisor/ovm3/resources/Ovm3HypervisorGuru.java

```diff
@@ -105,9 +105,6 @@ public boolean trackVmHostChange() {
                     EndPoint ep = endPointSelector.selectHypervisorHost(new ZoneScope(host.getDataCenterId()));
                     host = hostDao.findById(ep.getId());
                     hostDao.loadDetails(host);
-                    // String snapshotHotFixVersion = host.getDetail(XenserverConfigs.XS620HotFix);
-                    // if (snapshotHotFixVersion != null && snapshotHotFixVersion.equalsIgnoreCase(XenserverConfigs.XSHotFix62ESP1004)) {
-                    return new Pair<Boolean, Long>(Boolean.TRUE,  Long.valueOf(ep.getId()));
```

Comment:
as we never return anything but ew Pair<Boolean, Long>(Boolean.FALSE, Long.valueOf(hostId)) we might as well return it as the first line of the method. There is side effect in the code that is needed, so let's extract that to another (set of) private method(s). improves readability and thus maintainability of the code